### ISSUE TITLE
The "Restore" button in the chart admin does not work

### ISSUE THREAD

**Issue by pabloarosado** [owid/owid-grapher#3643]
## Description

The "Restore" button in the grapher chart admin "Revisions" tab doesn't work. The "Compare to previous" button right next to it does work though.

## Expected behaviour

The "Restore" button should restore a different version of the chart in the preview.

## Steps to reproduce

1. Go to any chart admin (for example [this one](https://admin.owid.io/admin/charts/3450/edit)).
2. Open the "Revisions" tab.
3. Click on various "Restore" buttons and see that the preview doesn't change (even if, when clicking on "Compare to previous" you see that there were changes).

## Screenshots

https://github.com/owid/owid-grapher/assets/12246978/bfff5c4b-cba8-4957-b22d-b98661dd3cda

**Comment by sophiamersmann** [comment]
Yes, I'll have a look (but may re-assign it to you if a fix is not obvious to me)

**Comment by rakyi** [comment]
It seems the fix doesn't work for older versions. Is it worth fixing further?
